.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+  "name": "Node.js & TypeScript",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:0-20",
+  "features": {
+    "ghcr.io/devcontainers/features/github-cli:1": {},
+    "ghcr.io/devcontainers/features/rust:1": {},
+    "ghcr.io/devcontainers/features/ruby:1": {}
+  },
+  "overrideFeatureInstallOrder": ["ghcr.io/devcontainers/features/rust"]
```

Comment:
For my info: The docs aren't specific on this feature, by just supplying rust, does that bump it above the others and they will run afterwards?

Reply:
Yes. https://github.com/devcontainers/spec/blob/c0beafa8f08db7df99ca12af9d407fb6c9b5f8db/docs/specs/devcontainer-features.md?plain=1#L270

Reply:
Weird that this isn't propagated to their [JSON reference](https://containers.dev/implementors/json_reference/). Thanks for that!